company/values.md

```diff
@@ -1,15 +1,34 @@
 # Sourcegraph values
 
-Our values are:
+Our values reflect who we are and how we work.
 
-## People
+## People first
 
-Together we are advancing technology for the good of people all around the world. We will attract, hire and retain the best teammates in the world and treat everyone in a first-class manner.
+[Our amazing team](teammate_qualities.md) is what makes Sourcegraph special. Together we are advancing technology for the good of people all around the world and we treat everyone in a first-class manner.
 
-## Journey
+## Customer driven progress
 
-The collection of moments, experiences, and memories that the team shares as we make progress: the light moments, the laughter, the team coming together to solve a problem.
+We have a [vision](strategy.md#vision) to build a product that is loved and valued by every company and developer. We learn what our customers want by understanding the challenges that they face, iterating quickly, and soliciting feedback. We feel good when we help other people, we are passionate about developer tools, and our business succeeds when customers are happy to pay for the value that we provide.
 
-## Progress
+Being customer driven does not mean we build bespoke features. Doing so would be a very inefficient and costly way to achieve our vision. Instead, we use customer feedback as data to help us identify work that would be valuable to a broad set of customers, and then directly engage with specific customers to validate that work.
 
-The continuous march toward achieving our mission as a team: the milestones, the successes, and the breakthroughs.
\ No newline at end of file
+## Open and transparent
```

Comment:
Is there still room for the phrase "open company" here somewhere? I like the phrase because it illustrates that openness is not just one of our many values, it's at the core of the company.